.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```

Comment:
`fast_finish` was [blog-announced here](http://blog.travis-ci.com/2013-11-27-fast-finishing-builds/) and documented here](http://docs.travis-ci.com/user/build-configuration/#Fast-finishing).

Only [`PHP 7'](http://en.wikipedia.org/wiki/PHP#Release_history) is announced, and IMHO `nightly`release support do not make sense for a business code library. So no need to add `7.1` so far.

Since we're using more tiers, dev-related libraries, it is quite convenient to have all build processes details in only one go, in order to check what's going on: if any build will finish as soon as a job has failed, this wouldn't ease the debug process (see #20 and its [related Travis output](https://travis-ci.org/ronanguilloux/IsoCodes/builds/59845133) by example).

So I propose:
- no `fast_finish` so far
- no `7.1` CI tests so far

@TomasVotruba: Please consider the PHPUnit question below ;-)